starter-code/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,27 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { MyHomeComponent } from './my-home/my-home.component';
+import { MyMovieComponent } from './my-movie/my-movie.component';
+
+
+const routes: Routes = [
+  // router.get('/', ...
+  // { path: '', component: LandingPageComponent },
+
+  // router.get('/contacts/:blahId', ...
+  // { path: 'contacts/:blahId', component: ContactDetailsComponent },
+
+  // router.get('/contacts/:blahId', ...
+  { path: '', component: MyHomeComponent },
```

Comment:
I guess you are right about the home route. But you could also create a handler for `path:""` and just redirect the user to `path:"home"`, if you wanted to